### ISSUE TITLE
HTML order. Input is last.

### ISSUE THREAD

**Issue by igoriok1994** [markusenglund/react-switch#70]
Hi, 

Can you check HTML input's order like that.

![image](https://user-images.githubusercontent.com/37294916/77044274-cacc0c80-69c7-11ea-89b0-55cfcdf6cc79.png)

It will be better, as I could properly use CSS tidle (~) selector to control handle.

Something like that: 

```
input:focus ~ .react-switch-handle {........}
```

**Comment by igoriok1994** [comment]
Or maybe you can make callbacks for onFocus and onBlur events?

**Comment by igoriok1994** [comment]
Also, why input ref name is **$inputRef** for dev env and **y** for production? 
How can I access input ref?
ReactDOM.findDOMNode is not a good practice.

**Comment by markusenglund** [comment]
No, I won't make that change. The invisible input element acts in lots of strange ways on different browsers and platforms. Changing the order might very well break something.

I didn't have in mind that someone would try to access the `inputRef`. As you noticed I've minified it in the production mode to save bytes.

Sorry for not responding earlier.
